test/conftest.py

```diff
@@ -46,7 +46,11 @@ def __init__(self):
 
     @property
     def logs(self):
-        return self.driver.execute_script("return window.logs")
+        logs = self.driver.execute_script("return window.logs")
+        return '\n'.join(str(x) for x in logs)
```

Comment:
This makes it harder to assert that an entire line appears exactly in the log.  See `test_python.py::test_init`, for example.

I think this should remain as returning a list, but you could add a `logs_as_string` helper function.

Reply:
I was wondering about that. Will create a separate helper function.

The question is though how often we will match a substring as opposed to a full line. If more often, `some_line in selenium.logs.splitlines()` might be not so bad, compared to,
```py
some_substr in selenium.logs_as_string
# together with
some_line in selenium.logs
```
I am more used for logs to be a string  rather than a list (even if it means doing back and forth conversions internally) but it is indeed subjective..